scripts/webpack.config.dev.js

```diff
@@ -7,12 +7,15 @@ const genConfig = ({
   entry, filename, library, libraryTarget,
 }) => ({
   ...common,
+  target: 'web',
   mode: 'development',
+  devtool: 'inline-source-map',
   entry,
   output: {
     filename,
     library,
```

Comment:
This is the only place where the two submodules are connected. I'ld like to fix that, eventually.